TileFormats/Instanced3DModel/README.md

```diff
@@ -98,19 +112,54 @@ In either case, `header.gltfByteLength` contains the length of the glTF field in
+| `position.z` | `uint16` | The z-coordinate in quantized cartesian coordinates. | `yes` |
+| `v1.x` | `uint` | The first component of the oct-encoded vector `v1`. | `yes` |
+| `v1.y` | `uint` | The second component of the oct-encoded vector `v1`. | `yes` |
+| `v2.x` | `uint` | The first component of the oct-encoded vector `v2`. | `yes` |
+| `v2.y` | `uint` | The second component of the oct-encoded vector `v2`. | `yes` |
+| `batchId` | `uint16` | ID in the range `[0, length of arrays in the Batch Table)`, which indicates the corresponding properties. | `if header.batchTableByteLength > 0`
+
+### X, Y, and Z for Translation
+
+`x`, `y`, and `z` are stored as `uint16` positions in the quantized instance region defined by the `origin` and `span` fields in the header.
+
+**Figure 2:** The instance region defined by `origin` and `span`
+
+![](figures/instance-region.png)
+
+Transforming `position` in instance region space to `position_w` in world space can be done using the formula:
+
+`position_w` = `origin` + [`position` * `span` / (`2^16-1`)].
+
+### V1, and V2 for Rotation
+
+`v1` and `v2` are stored as two components in oct-encoded format as described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/)
+The [AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module in Cesium contains an implementation for encoding and decoding vectors in this fashion.
+
+In their decoded form, `v1` and `v2` establish a new orthonormal basis, effectively rotating the coordinate system.
+
+The `y` and `x` vectors in the natural basis are transformed to map onto `v1` and `v2` respectively.
+
+**Figure 3:** A 3D box in the natural basis
+
+![](figures/box-unit-basis.png)
 
-|Field name|Data type|Description|
-|----------|---------|-----------|
-| `longitude` | `double` | The longitude, in radians, in the range `[-PI, PI]`. |
-| `latitude` | `double` | The latitude, in radians, in the range `[-PI / 2, PI / 2]`. |
-| `batchId` | `uint16`  | ID in the range `[0, length of arrays in the Batch Table)`, which indicates the corresponding properties. |
+**Figure 4:** A 3D box in the rotated basis
 
-_TODO: make this much more memory efficient and more general, [#33](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/33)._
+![](figures/box-rotated-basis.png)
 
-When `header.batchTableByteLength` is zero, which indicates there is not a batch table, `batchId` is omitted, so each instance contains only `longitude` and `latitude` fields.
+The mapping for `z` can be omitted since it will be the cross product of `v1` and `v2`.
 
-Each instance is in the east-north-up reference frame (`x` points east, `y` points north, and `z` points along the geodetic surface normal).
+A rotation matrix can be created from `v1`, `v2`, and `v3`=`v2`x`v1` by placing `v2` into the first column,
```

Comment:
Can this include simple equations like, for example, https://github.com/tsturm/glTF/blob/master/extensions/Vendor/FRAUNHOFER_materials_pbr/Appendix.md#specular-brdf